src/Confluent.Kafka/Message.cs

```diff
@@ -18,6 +18,9 @@
 
 namespace Confluent.Kafka
 {
+    /// <summary>
+    ///     Represetns a (deserialized) Kafka Message - what, where and when.
```

Comment:
what: key, value
where: topic, partition, offset
when: timestamp

I'll get rid of that for now and describe more fully in a later iteration.

typo fixed.